src/pages/resultsView/cancerSummary/CancerSummaryChart.tsx

```diff
@@ -748,9 +776,11 @@ export class CancerSummaryChart extends React.Component<
                 </div>
                 <DownloadControls
                     getSvg={() => this.svg}
+                    getData={(dataType: DataType) => this.getData(dataType)}
```

Comment:
can it just be `getData={this.getData}`?

Reply:
this is updated.